content/en/observability_pipelines/_index.md

```diff
@@ -43,31 +43,80 @@ Datadog recommends you update Observability Pipelines Worker (OPW) with every mi
-- Sensitive data redaction: Remove sensitive data from your logs before they are routed outside of your infrastructure.
+The Observability Pipelines Worker is the software that runs in your infrastructure. It centrally aggregates, processes, and routes your logs based on your use case. This means you can redact sensitive data, pre-process logs, and determine which destinations they should go to, all before the logs leave your environment.
 
-The Observability Pipelines Worker is the software that runs in your infrastructure. It aggregates and centrally processes and routes your logs based on the selected use case.
-
-The Datadog UI provides a control plane to manage your Observability Pipelines Workers. You can build and edit pipelines, deploy pipeline changes to your Workers, and monitor your pipelines to evaluate the health of your pipelines.
+The Observability Pipelines UI provides a control plane to manage your Observability Pipelines Workers. You build and edit pipelines and deploy pipeline changes to your Workers from there. You can also enable out-of-the-box monitors for your pipelines so that you can evaluate their health.
 
 ## Get started
 
+To set up a pipeline:
+
 1. Navigate to [Observability Pipelines][1].
-1. Select a use case:
+1. Select a template:
     - [Log volume control][2]
     - [Dual ship logs][3]
     - [Split logs][4]
     - [Archive logs to Datadog Archives][5]
     - [Sensitive data redaction][6]
-    - [Log Enrichment][7]
-1. Enable monitors.
+1. Select and set up your source.
+1. Select and set up your destinations.
+1. Set up you processors.
+1. Install the Observability Pipelines Worker.
+1. Enable monitors for your pipeline.
+
+See [Advanced Configurations][7] for bootstrapping options and for details on setting up the Worker with Kubernetes.
+
+## Explore Observability Pipelines
+
+### Build pipelines with out-of-the-box templates
+
+{{< img src="observability_pipelines/templates.png" alt="The Observability Pipelines UI showing the six templates" style="width:100%;" >}}
+
+The [templates](#out-of-the-box-templates) are purpose-built for the following use cases:
+
+#### Log Volume Control
+
+Raw logs are noisy and only some logs are useful for further search and analysis during investigations. Use the Log Volume Control template to determine which logs to send to your indexed solution, such as a SIEM or log management solution. This helps you to increase the value of your indexed logs and also remain within your planned budget.
+
+#### Dual Ship Logs
+
+As your organization grows, your observability needs for different use cases, such as security, archiving, and log management also change. This could mean having to trial different archiving, SIEM, and log management solutions. However, managing log pipelines to different solutions can get complicated. Use the Dual Ship Logs template to centrally aggregate, process, and send copies of your logs to different destinations.
```

Comment:
```suggestion
As your organization grows, your observability needs for different use cases, such as security, archiving, and log management, also change. This could mean having to trial different archiving, SIEM, and log management solutions. However, managing log pipelines to different solutions can be complicated. Use the Dual Ship Logs template to centrally aggregate, process, and send copies of your logs to different destinations.
```